streams/src/test/java/org/apache/kafka/streams/tests/StreamsUpgradeTest.java

```diff
@@ -146,30 +146,22 @@ public ByteBuffer subscriptionUserData(final Set<String> topics) {
             // 2. Task ids of previously running tasks
             // 3. Task ids of valid local states on the client's state directory.
             final TaskManager taskManager = taskManger();
-
-            final Set<TaskId> standbyTasks = taskManager.tasksOnLocalStorage();
-            final Set<TaskId> activeTasks = prepareForSubscription(taskManager,
-                                                                   topics,
-                                                                   standbyTasks,
-                                                                   REBALANCE_PROTOCOL);
```

Comment:
req: After this deletion, `REBALANCE_PROTOCOL` defined on line 66 is not used anymore and can be deleted, too.